src/includes/set-tag/android.mdx

```diff
@@ -0,0 +1,3 @@
+```java
+Sentry.setTag("page_locale", "de-at");
```

Comment:
I'm assuming this actually works? Where does Sentry come from? Why is it SentryAndroid in other examples? Do we need to show them import statements?

Reply:
this works, `Sentry` is the static class that has the available API.
SentryAndroid is just to init the SDK manually, passing the Android context which is dependent on the Android framework